src/help/fabs_validations.csv

```diff
@@ -0,0 +1,28 @@
+Relates to Files,Rule Name,Rule Detail,Severity,Status,Type
+D2,D1,"Element value must be a valid enumeration as described in the ""Domain_Values.xlsx"" spreadsheet.",n/a,Implemented in ASP,Requirement

```

Comment:
We are aware that all of these aren't accurately named, labeled, or written correct?

Reply:
These were copy pasted/taken from the existing rules. Will double check but I think this is it

Reply:
Words match up as far as I can tell with the provided table
https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/dev/src/help/validations.csv

Reply:
Just making sure because things like `Must be blank for aggregate records; required for non-aggregate records.` have no meaning whatsoever and most if not all of these don't line up to the current rules